plugins/linear-genome-view/src/LinearGenomeView/components/Search.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useRef, useState } from 'react'
+import { getSession } from '@jbrowse/core/util'
+import { observer } from 'mobx-react'
+import SearchIcon from '@material-ui/icons/Search'
+import TextField from '@material-ui/core/TextField'
+import { useTheme } from '@material-ui/core/styles'
+import { fade } from '@material-ui/core/styles/colorManipulator'
+import { LinearGenomeViewModel } from '..'
+
+const WIDGET_HEIGHT = 32
+const SPACING = 7
+
+export default observer(({ model }: { model: LinearGenomeViewModel }) => {
+  const [value, setValue] = useState<string | undefined>()
+  const inputRef = useRef<HTMLInputElement>(null)
+  const theme = useTheme()
+  const { coarseVisibleLocStrings: visibleLocStrings } = model
+  const session = getSession(model)
+
+  function navTo(locString: string) {
+    try {
+      model.navToLocString(locString)
+    } catch (e) {
+      console.warn(e)
+      session.notify(`${e}`, 'warning')
+    }
+  }
+
+  return (
+    <form
+      data-testid="search-form"
+      onSubmit={event => {
+        event.preventDefault()
+        inputRef && inputRef.current && inputRef.current.blur()
+        value && navTo(value)
+
+        // need to manually call the action of blur here
+        setValue(undefined)
```

Comment:
note that the onBlur was not triggered by the above inputRef.current.blur() so i manually call setValue(undefined) here